<launch>
<!--
  <node pkg="keyboard_control" exec="keyboard_control" name="keyboard_control" output="screen" launch-prefix="xterm -e">
    <param name="linear_v" value="5.0" /> 
    <param name="angular_v" value="5.0" /> 
    <param name="publish_topic" value="/PioneerP3DX/cmd_vel" />
  </node>
-->

  <node pkg="keyboard_control" exec="keyboard_control_plus" name="keyboard_control" output="screen" launch-prefix="xterm -e">
    <param name="linear_v_inc" value="0.1" />
    <param name="angular_v_inc" value="0.3" />
    <param name="publish_topic" value="/PN1wxofz/cmd_vel" />
  </node>
</launch>
